src/xrCDB/xrCDB.cpp

```diff
@@ -14,6 +15,8 @@ namespace Opcode
 using namespace CDB;
 using namespace Opcode;
 
+static command_line_key<bool> mt_cdb("-mt_cdb", "mt_cdb", false);
```

Comment:
Ideally, the name of the key should be specified without `-` part. (`"mt_cdb"` instead of `-mt_cdb`)
And let the code in command_line_key.cpp check for that `-` automatically.

Reply:
ok, so this line should actually look like `static command_line_key<bool> mt_cdb("mt_cdb", "mt_cdb", false);`?
There are options in the engine that actually do not start with a `-` (i think mostly for debug stuff).